Packs/GSuiteAdmin/ReleaseNotes/1_2_0.json

```
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes": "Updated the commands gsuite-custom-user-schema-create and gsuite-custom-user-schema-update, now the argument 'field_json_entry_id' is getting an entry ID and will read the content of the json file in this entry id.\nUpdated the command gsuite-datatransfer-request-create, now the argument 'applications_raw_json_entry_id' is getting an entry ID and will read the content of the json file in this entry id."}
```

```suggestion
{"breakingChanges":true,"breakingChangesNotes": null}
```
basically you can take the updated version of the breaking changes above, but leaving the notes empty is also ok as it will eventually take the full RN which also specify the breaking changes